api/meetings/permissions.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import permissions
+
+from api.base.utils import assert_resource_type
+from osf.models import AbstractNode
+
+
+class IsPublic(permissions.BasePermission):
+    """
+    Only returning public nodes as meeting submissions.
+    """
+
+    acceptable_models = (AbstractNode,)
+
+    def has_object_permission(self, request, view, obj):
+        assert_resource_type(obj, self.acceptable_models)
+        return obj.is_public
```

Comment:
I just matched existing functionality! Projects don't show up on current meeting submissions page, even if they're your own, unless they're public.

Reply:
Ah, collections doesn't do this because they're writeable from the API, so all of their permissions are more complicated. The functionality makes sense, but it's different that other places and I don't know how many other people knew that, so I'll definitely try to get that documented.